<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureRoom</class>
 <widget class="QDialog" name="ConfigureRoom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>551</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure room</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabsConfiguration">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Affiliations</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="labelLoading">
           <property name="minimumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16</width>
             <height>16</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>0</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="1">
        <widget class="QPushButton" name="pushAddJid">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="pushRemoveJid">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Remove</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" rowspan="4">
        <widget class="QTreeWidget" name="treeAffiliations">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
         <property name="allColumnsShowFocus">
          <bool>true</bool>
         </property>
         <column>
          <property name="text">
           <string notr="true">JID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Reason</string>
          </property>
         </column>
         <item>
          <property name="text">
           <string>Owners</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Administrators</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Members</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Banned</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Other</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="margin">
        <number>0</number>
       </property>
       <item row="0" column="0">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>525</width>
            <height>273</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabsConfiguration</tabstop>
  <tabstop>treeAffiliations</tabstop>
  <tabstop>pushAddJid</tabstop>
  <tabstop>pushRemoveJid</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigureRoom</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigureRoom</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>304</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushAddJid</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureRoom</receiver>
   <slot>slotAddJid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>587</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushRemoveJid</sender>
   <signal>clicked()</signal>
   <receiver>ConfigureRoom</receiver>
   <slot>slotRemoveJid()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>588</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>treeAffiliations</sender>
   <signal>currentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</signal>
   <receiver>ConfigureRoom</receiver>
   <slot>slotCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>209</y>
    </hint>
    <hint type="destinationlabel">
     <x>641</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotCurrentTabChanged(int)</slot>
  <slot>slotAddJid()</slot>
  <slot>slotRemoveJid()</slot>
  <slot>slotCurrentItemChanged(QTreeWidgetItem*,QTreeWidgetItem*)</slot>
 </slots>
</ui>
